proximo-server/amqp.go

```diff
@@ -102,3 +103,18 @@ func (h *amqpHandler) HandleConsume(ctx context.Context, consumer, topic string,
 func (h *amqpHandler) HandleProduce(ctx context.Context, topic string, forClient chan<- *Confirmation, messages <-chan *Message) error {
 	panic("not implemented")
 }
+
+func (h *amqpHandler) Status() (bool, []error) {
```

Comment:
I'm not really a fan of returning an array of errors. Would it be worth creating a new error type instead and just changing this to return `(bool, error)`

Reply:
@ggordan I supose we can do it :)